docs/release/release-2.0.rst

```diff
@@ -10,6 +10,7 @@ New Features
 * Change the default configuration directory to ``~/.prestoadmin`` and the
   default log directory to ``~/.prestoadmin/log``.
 * Remove the requirement for running and installing presto-admin with sudo.
```

Comment:
suggest:
"Remove the requirement for running and installing the `presto-admin` program with sudo. Sudo access is still required for all Presto nodes, for example to install RPMs.